libs/api/data-transfer/src/lib/models/notification.model.ts

```diff
@@ -0,0 +1,17 @@
+import { OrderStatusChangedDto } from '../dto';
+import { UserModel } from './user.model';
+
+export enum NotificationType {
+  STATUS_CHANGED = 'ORDER_STATUS_CHANGED',
+  INVITATION_SENT = 'INVITATION_SENT',
+}
+
+export type NotificationAction = `${NotificationType}`;
+
+export interface NotificationModel {
+  id: string;
+  action: NotificationAction;
+  data?: OrderStatusChangedDto;
```

Comment:
I bet it should be unknown instead of OrderStatusChangedDto. You should map data based on action